packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -19,7 +19,6 @@ data class GutenbergProps @JvmOverloads constructor(
     val translations: Bundle,
     val isDarkMode: Boolean,
     val htmlModeEnabled: Boolean,
-    val enableEditorOnboarding: Boolean,
     var firstGutenbergEditorSession: Boolean
```

Comment:
I opted to retain the `firstGutenbergEditorSession` property even though it is currently unused, as I figured persisting this information may prove useful for any future onboarding features. By retaining it, we have this flag property disabled for non-first-time users whenever we add an onboarding feature. However, we could remove the logic from both platforms if others see it differently.